lib/commons/dom/is-skip-link.js

```diff
@@ -54,3 +35,22 @@ export default function isSkipLink(element) {
     element.DOCUMENT_POSITION_FOLLOWING
   );
 }
+
+function generateFirstPageLink() {
+    let firstPageLink;
```

Comment:
```suggestion
  let firstPageLink;
```

Reply:
Strange, I guess my prettier didn't work on this commit.